airbyte-integrations/connectors/source-linkedin-ads/source_linkedin_ads/source.py

```diff
@@ -336,6 +336,63 @@ class AdCreativeAnalytics(LinkedInAdsAnalyticsStream):
     pivot_by = "CREATIVE"
 
 
+class AdCreativeName(Creatives):
+
+    endpoint = "adCreativesV2"
```

Comment:
No need to re-declare the endpoint var here, since it's a part of Creatives class